c/unixFileService.c

```diff
@@ -920,6 +920,38 @@ static int serveUnixFileMetadata(HttpService *service, HttpResponse *response) {
   return 0;
 }
 
+static int serveUnixFileChangeOwner (HttpService *service, HttpResponse *response) {
+  HttpRequest *request = response->request;
+  char *routeFileFrag = stringListPrint(request->parsedFile, 2, 1000, "/", 0);
+  char *encodedRouteFileName = stringConcatenate(response->slh, "/", routeFileFrag);
+  char *routeFileName = cleanURLParamValue(response->slh, encodedRouteFileName);
+
+  char *userId    = getQueryParam(response->request, "user");
+  char *groupId   = getQueryParam(response->request, "group");
+  char *pattern   = getQueryParam(response->request, "pattern");
+  char *recursive = getQueryParam(response->request, "recursive");
+
+  if (!strcmp(request->method, methodPOST)) {
+    directoryChangeOwnerAndRespond (response, routeFileName,
+                          userId, groupId, recursive, pattern);
+  }
+  else {
+    jsonPrinter *out = respondWithJsonPrinter(response);
+
+    setResponseStatus(response, 405, "Method Not Allowed");
+    setDefaultJSONRESTHeaders(response);
+    addStringHeader(response, "Allow", "POST");
+    writeHeader(response);
+
+    jsonStart(out);
```

Comment:
Do we really need this empty JSON? If not, we can get rid of the `respondWithJsonPrinter` call.

Reply:
The jira ticket specifies that we support the POST for this feature.  If you want to remove the message indicating what the is error in the request, then we can remove it.  Otherwise, I would keep it in.

Reply:
I was talking about the lines:
```c
jsonStart(out);
jsonEnd(out);
```

Reply:
I can remove this is un-necessary.  Make sure it is;  there are 7 other places in this file with the exact code sequence.

Reply:
You don't need to fix everything. If this is not necessary, why add more of it.